test/metabase/channel/shared_test.clj

```diff
@@ -0,0 +1,33 @@
+(ns metabase.channel.shared-test
+  (:require
+   [clojure.test :refer :all]
+   [metabase.channel.shared :as channel.shared]
+   [metabase.test :as mt]))
+
+(deftest cron-to-friendly-description-test
+  (mt/with-dynamic-fn-redefs [channel.shared/schedule-timezone (constantly "UTC")]
```

Comment:
Not worth testing with another timezone as well, like before?

Reply:
oh yea, lemme add one.